calculator.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+ 
+all_data = {
+    "Transaction Type": "",
+    "Origin or Destination": "",
```

Comment:
this differs between income vs expense, you'll want to add this key-value pair dynamically